spec/mailers/user_mailer_spec.rb

```diff
@@ -81,7 +81,7 @@
 
       context 'body includes' do
         it 'password reset url' do
-          expect(mail.body.raw_source).to include spree.edit_spree_user_password_url
+          expect(mail.html_part.body).to include spree.edit_spree_user_password_url
```

Comment:
We should probably test both parts because some people use email clients that display only the text version.

Reply:
Oh, that's a good point!

I took a look and found that now that there's two templates, the email is generated with two parts:
```
mail.parts
=> [#<Mail::Part:48320, Multipart: false, Headers: <Content-Type: text/plain>, <Content-Transfer-Encoding: 7bit>>,
 #<Mail::Part:48340, Multipart: false, Headers: <Content-Type: text/html>, <Content-Transfer-Encoding: quoted-printable>>]
```

So we can test both of those parts. `raw_source` no longer works because it's a multi-part email. So we can test like this:
```suggestion
          expect(mail.text_part.body).to include spree.edit_spree_user_password_url
          expect(mail.html_part.body).to include spree.edit_spree_user_password_url
```